<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanWorkPieceDialog</class>
 <widget class="QDialog" name="ScanWorkPieceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScanWorkPieceDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="starScanButton">
       <property name="text">
        <string>StartScan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopScanButton">
       <property name="text">
        <string>StopScan</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="currentSerialNumbersLabel">
       <property name="text">
        <string>current serial numbers:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="currentSerialNumberslineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="currentTotalValueLabel">
       <property name="text">
        <string>current total num:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="targetNumLabel">
       <property name="text">
        <string>target num:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="targetValueLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
